### PR TITLE
Fixes #205 Add top menu links on right on contact and terms page

### LINKED ISSUE #205: Improve top menu on pages and add links
Please match the design of pages like http://susper.com/about to http://loklak.net/about and:
* add top menu links (Home, About, Contact) to the following pages: Terms, About, Contact
* Align logo and text on these pages same as loklak.net
* Ensure you keep same distance to left and right as loklak.net

### PR DESCRIPTION
**Pull request by harshit98**
Resolves #205 

Things I have done in this PR : 
- Added top menu links on top-right corner on contact and terms page.

Screenshots : 

Terms Page

selection_003

Contacts Page 

selection_002

### CODE REVIEW

File: src/app/contact/contact.component.html [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
   <div class="container-fluid">
     <div class="navbar-header">
-      <a class="navbar-brand about-navbar" href="//susper.com">
+      <a class="navbar-brand contact-navbar" href="//susper.com">
         <img alt="brand" class="navbar-logo" src="../../assets/images/susper.svg">
       </a>
     </div>
-    <p class="navbar-text navbar-right">Contact Us</p>
+    <p class="navbar-collapse navbar-right">
+      <a href="terms">Terms</a>
```
Comment (nikhilrayaprolu) [inline src/app/contact/contact.component.html]: this is a wrong way to navigate to a page. this will reload the whole page use `routerLink`

Comment (harshit98) [inline src/app/contact/contact.component.html]: @nikhilrayaprolu ok. I'll do the changes.

**Comment by harshit98** [comment]
@nikhilrayaprolu done!

What I did : 
- changed `href` to `routerLink`
- solved responsiveness issue for terms and contact page as well.

Screenshot : 

About Page 

selection_009

Terms page 

selection_010

Contact page

selection_011

**Comment by harshit98** [comment]
@mariobehling We can open up a separate issue, regarding to responsiveness of top menu. 

At present,  this is how loklak works on small screens : 

selection_025

selection_026

Should we proceed following this design?

**Comment by mariobehling** [comment]
Ok, sounds good for me.

I opened an issue here: https://github.com/fossasia/susper.com/issues/249

LGTM

**Pull request merged**